docs/details/integration/os/buildroot/quick_setup.rst

```diff
@@ -0,0 +1,55 @@
+Quick Setup 
+###########
+
+A Git repository is available that includes everything needed to test the 
+Buildroot setup without following the guide. It is intended for testing 
+purposes and can be used if you are already familiar with Buildroot concepts 
+and architecture.
+
+Get the repository
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/lvgl/lv_buildroot.git
```

Comment:
Could you please add the link to this doc in lv_buildroot repo's README.md?

Reply:
Glad to hear that! 
I added a PR on lv_buildroot repository.
https://github.com/lvgl/lv_buildroot/pull/1